gwcs/wcs.py

```diff
@@ -411,3 +416,100 @@ def footprint(self, axes, center=True):
         # return self.output_coordinate_system.world_coordinates(result[:,0], result[:,1])
         # except:
         # return result
+
+    @classmethod
+    def from_fiducial(cls, fiducial, coordinate_frame=None, projection=None,
+                      transform=None, name='', domain=None):
+        """
+        Create a WCS object from a fiducial point in a coordinate frame.
+
+        If an additional transform is supplied it is prepended to the projection.
+
+        Parameters
+        ----------
+        fiducial : `~astropy.coordinates.SkyCoord` or tuple of float
+            One of:
+                A location on the sky in some standard coordinate system.
+                A Quantity with spectral units.
+                A list of the above.
+        coordinate_frame : ~gwcs.coordinate_frames.CoordinateFrame`
+            The output coordinate frame.
+            If fiducial is not an instance of `~astropy.coordinates.SkyCoord` it
```

Comment:
I think you meant  "If fiducial **is** ~~not~~ an instance of `~astropy.coordinates.SkyCoord` it should be specified here." At least this is what I understand the code is doing on line 482.